pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -1056,6 +1056,21 @@ func (m *ManagerImpl) isDevicePluginResource(resource string) bool {
 	return false
 }
 
+// GetAllDevices returns information about all the devices known to the manager
+func (m *ManagerImpl) GetAllDevices() map[string]map[string]pluginapi.Device {
```

Comment:
If here will be not a  pluginapi.Device, but appropriate version of podresourcesapi.ContainerDevices, like in GetDevices, I think, you could avoid dependencies in cm and podresources packages. I don't see extra field in Device, which doesn't exists in ContainerDevices.

Reply:
Good point Alexey! Don't see it as a major blocker for the feature though, will address this in a follow-up PR.